### ISSUE TITLE
Help finding the dataset

### ISSUE THREAD

**Issue by arundasan91** [rkosti/emotic#1]
Hello @rkosti ,

Could you please share a link to the dataset or a repo which hosts it ? 

Thanks,
Arun

**Comment by arundasan91** [comment]
Hi @rkosti ,

I got [the link.](http://sunai.uoc.edu/emotic/download.html) Thanks a lot. Good work on the paper 👍 

All the best for your future research.

**Comment by rkosti** [comment]
Hi @arundasan91, 
Thanks for the comments. 
The labels were not hosted earlier, now they are available to download and use. The models used will be added soon.

**Comment by arundasan91** [comment]
@rkosti , Brilliant work! Congrats. Is there a python version of reading annotations?

**Comment by rkosti** [comment]
Thanks!
No, and I don' think we will. We use Torch7 for our work and we will release training code and models in lua and TensorFlow.  
However, If you do make one in python, please let us know so that we can add it along with the matlab version.

**Comment by arundasan91** [comment]
Sure thing. Thanks for the info. I was trying to load the annotation mat file to pandas since I don't have matlab installed locally. Will update you if I get a nested dictionary data structure of the matlab file ready. I appreciate your help.

**Comment by GKalliatakis** [comment]
@arundasan91 Did you manage to load the annotation .mat file in python ?

**Comment by arundasan91** [comment]
@GKalliatakis , No. I did not. The project was put on hold. 

The annotations are of variable length and I had a tough time writing the code. I later used the Matlab scripts to explore the dataset. I tried exporting the mat file to csv but that didn't work either. 

I found this recently in Matlab code `demo_single.m`:

```
% NOTE: train, test, and val have almost the same structure. The only difference
% is that val contains up to 10 annotations per sample (each sample has been
% annotated by up to 10 different annotators), test contains up to 6 annotations per sample (each sample has been
% annotated by up to 6 different annotators), and train contains just 2
% annotators per image (one for continuous dimensions and one for
% categories)
```
This is something which would've helped. I was trying to find the number of columns dynamically.

**Comment by missyun125** [comment]
I can't download EMOTIC dataset on the official website. Can you send the dataset to my email（604133530@qq.com）?

**Comment by missyun125** [comment]
@GKalliatakis  I can't download EMOTIC dataset on the official website. Can you send the dataset to my email（604133530@qq.com）?

**Comment by Tandon-A** [comment]
@missyun125 

It seems that the access to the dataset is changed. You now need to fill the [ access form](https://docs.google.com/forms/d/e/1FAIpQLScXwxhEZu7RpHwgiRqVfb09GzHSSyIm64hJQMgHSLm75ltsFQ/viewform) first. After reviewing the details, Ronak would you send the link to the dataset. 

Regards, 
Abhishek
